image.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Image</title>
+    <!-- <style>
+        div.p {
+            text-align: left;
+        }
+        
+        div.a {
+            text-align: right
+        }
+    </style> -->
+</head>
+
+<body>
+    <div>
+        <table style="text-align:center">
+            <tr>
+                <td height="50%" width="50%">
+                    <p style="text-align:justify">Web development is the work involved in developing a Web site for the Internet (World Wide Web) or an intranet private network).[1] Web development can range from developing a simple single static page of plain text to complex web
+                        applications, electronic businesses, and social network services. A more comprehensive list of tasks to which Web development commonly refers, may include Web engineering, Web design, Web content development, client liaison, client-side/server-side
+                        scripting, Web server and network security configuration, and e-commerce development. Among Web professionals, "Web development" usually refers to the main non-design aspects of building Web sites: writing markup and coding.[2]
+                        Web development may use content management systems (CMS) to make content changes easier and available with basic technical skills</p>
+                </td>
+                <td height="50%" width="50%">
```

Comment:
provide a image tag inside the anchor tag

Reply:
okay I'll do it